.travis.yml

```diff
@@ -4,13 +4,28 @@ os:
   - osx
 julia:
   - 0.6
+  - 1.0
   - nightly
 matrix:
     allow_failures:
     - julia: nightly
 notifications:
   email: false
-# uncomment the following lines to override the default test script
-#script:
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("MacroTools"); Pkg.test("MacroTools"; coverage=true)'
+
+env:
+  global:
+    - DOCUMENTER_DEBUG=true
```

Comment:
Is it right to leave this in when merging to master?

Reply:
It depends on what you prefer. The configuration on discourse has the debug option on.